.travis.yml

```diff
@@ -2,6 +2,9 @@ sudo: required
 dist: trusty
 services:
   - docker
+branches:
+  only:
+    - publications
```

Comment:
This would prevent the entire Travis job from running except on the publications branch.